src/modules/products/ui/components/tags-filter.tsx

```
@@ -0,0 +1,72 @@
+                onCheckedChange={() => onClick(tag.name)}
+              />
+            </div>
+          ))
+        )
+      )}
+      {hasNextPage && (
+        <button
+          className="underline font-medium justify-start text-start disabled:opacity-50 cursor-pointer"
+          onClick={() => fetchNextPage()}
+          disabled={isFetchingNextPage}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)